webroot/results/includes/regions_get_current_records.php

```diff
@@ -7,21 +7,24 @@
     UNION
     " . regionsGetCurrentRecordsQuery( 'average', 'Average' ) . ") helper
   ORDER BY
-    rank, type DESC, year, month, day, roundTypeId, personName
+    helper.rank, helper.type DESC, helper.year, helper.month, helper.day, helper.roundTypeId, helper.personName
 ");
 
 function regionsGetCurrentRecordsQuery ( $valueId, $valueName ) {
   return
    "SELECT
       '$valueName'     type,
                        result.*,
-                       value,
+                       record.value,
       event.name       eventName,
       event.cellName   eventCellName,
-                       format,
+                       event.format,
       country.name     countryName,
       competition.cellName competitionName,
-                       rank, year, month, day
+                       event.rank,
+                       competition.year,
+                       competition.month,
+                       competition.day
```

Comment:
Nit: just to follow the convention used here, please align these to the left.

Reply:
I actually tried following the convention here for columns with aliases vs the original column names
![image](https://user-images.githubusercontent.com/32999689/64123460-1cef6c80-cdad-11e9-8136-f596eb81a51a.png)

Reply:
Ah! That looks weird, but let's leave it like that then ^^